nchelpers/__init__.py

```diff
@@ -982,10 +982,14 @@ def time_bounds_var_name(self):
 
         # If time:bounds attribute exists, use it.
         try:
-            return time_var.bounds
+            var_name = time_var.bounds
+            if hasattr(self.variables, var_name):
+                return var_name
         except AttributeError:
-            if self._cf_dataset_options['strict_metadata']:
-                return None
+            pass
```

Comment:
What happens when this falls through?

Reply:
It goes on to the next rule/heuristic. That's the structure of this method: a succession of attempts, which if they fail, fall through to the next attempt. This one is perhaps a little tricky: `AttributeError` is raised if there is no bounds attribute on `time_var`; if there is, then the existence of the claimed variable is tested. Failure on either test falls through to the next step.